tests/client.py

```diff
@@ -1465,6 +1528,27 @@ def update_engagements(self, engagement_id):
 
         return record
 
+    def update_tickets(self, ticket_id):
+        """
+        Hubspot API https://developers.hubspot.com/docs/api/crm/tickets
+        :params ticket_id: the pk value of the ticket record to update
+        :return:
+        """
+        url = f"{BASE_URL}/crm/v4/objects/tickets/{ticket_id}"
+
+        record_uuid = str(uuid.uuid4()).replace('-', '')[:20]
+        data = {
+            "properties": {
+                "subject": f"update record for testing - {record_uuid}"
```

Comment:
What's the purpose of sending this random `uuid` in the subject string ? Can't we use ticketID instead ?

Reply:
This is to get the new value every time we update the record. TicketID will always be constant.